cf_units/__init__.py

```diff
@@ -1128,7 +1128,7 @@ def is_unknown(self):
             False
 
         """
-        return self.category == _CATEGORY_UNKNOWN
+        return self.category == _CATEGORY_UNKNOWN or self.ut_unit is None
```

Comment:
@pelson Could `self.ut_unit` also be `_ud.NULL_UNIT`? If so, does this need to be included here in this check?

Reply:
The only way it can be NULL_UNIT is if the constructor succeeded and it is one of ``_NO_UNIT`` or ``_UNKNOWN_UNIT``.

We shouldn't consider "no unit" unknown, so the only case that it can be NULL_UNIT is if ``self.category == _CATEGORY_UNKNOWN``.

In summary, I don't think so.